dev-java/hamcrest-core/hamcrest-core-1.3-r2.ebuild

```diff
@@ -25,10 +25,10 @@ DEPEND="~dev-java/hamcrest-generator-${PV}:1.3
 
 RDEPEND=">=virtual/jre-1.8:*"
 
-JAVA_ANT_REWRITE_CLASSPATH="true"
-JAVA_ANT_CLASSPATH_TAGS="${JAVA_ANT_CLASSPATH_TAGS} java java-to-jar"
+JAVA_SRC_DIR="${PN}/src"
+JAVA_GENTOO_CLASSPATH="hamcrest-generator-1.3"
```

Comment:
from what we discussed, `hamcrest-generator` does not belong to classpath. also, it needs to be in `BDEPEND` instead of `DEPEND`.

Reply:
This was already changed.  Just adjusted typo from {P,B}DEPEND